### PR TITLE
Move Logo, Favicon, and Livereload to Config

### PR DESCRIPTION
**Pull request by codydaig**
In reference to #536 from @hawmen I moved the logo, favicon, and live reload options to the configuration files.

### CODE REVIEW

**Comment by lirantal** [comment]
thanks @codydaig 
LGTM.

File: config/env/default.js [hunk 1]
```diff
@@ -10,5 +10,7 @@ module.exports = {
 	port: process.env.PORT || 3000,
 	templateEngine: 'swig',
 	sessionSecret: 'MEAN',
-	sessionCollection: 'sessions'
+	sessionCollection: 'sessions',
+	logo: 'modules/core/img/brand/logo.png',
+	favicon: '/modules/core/img/brand/favicon.ico'
```
Comment (rhutchison) [inline config/env/default.js]: why is one file relative and the other absolute?  I know that you just took the value and put it in the config, but should this be switched to a relative path?  Templates follow relative paths.

File: config/env/development.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ module.exports = {
 		}
 	},
 	app: {
-		title: 'MEAN.JS - Development Environment'
+		title: 'MEAN.JS - Development Environment',
+        livereload: true
```
Comment (lirantal) [inline config/env/development.js]: livereload var should exist outside of app {}, move it beneath it

File: config/env/default.js (hunk 1 above)
Comment (codydaig) [inline config/env/default.js]: I copied and pasted the values. I'm not opposed to one over the other.

File: config/env/development.js (hunk 2 above)
Comment (codydaig) [inline config/env/development.js]: I don't have a compelling argument either way. Updated the spacing and moved it outside of app.

File: config/env/default.js (hunk 1 above)
Comment (codydaig) [inline config/env/default.js]: @rhutchison Is there a specific change you want me to make?

**Comment by lirantal** [comment]
@rhutchison  @codydaig  Ping me when you guys work it out to review and merge.

File: config/env/default.js (hunk 1 above)
Comment (rhutchison) [inline config/env/default.js]: @codydaig I know you just moved the value, but I think the project has been following mainly relative paths.  Can you change to relative and test?

**Comment by codydaig** [comment]
DO NOT MERGE! I messed up. I'll fix it after my next meeting.

**Comment by codydaig** [comment]
See #685 (this branch on my end is having git issues)

**Comment by lirantal** [comment]
Okie dokie

**Pull request closed without merging**